### PR TITLE
Update tidb-lightning-overview.md

### PR DESCRIPTION
**Pull request by leowangpingcap**
### What is changed, added or deleted? (Required)

adding all standard parquet files format support, document clearly that only data types in amazon aurora parquet are accepted,
this gives more confidence for user to use parquet files if they are not using amazon aurora parquet exported format

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
https://pingcap.zendesk.com/knowledge/editor/01FSTE2MZFGTVSR05PJ2PTME29/en-us?brand_id=360004187474

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by leowangpingcap** [comment]
@dveeden @db-will @shichun-0415 
merged into master as previous review was on 5.3

File: tidb-lightning/tidb-lightning-overview.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Currently, TiDB Lightning can mainly be used in the following two scenarios:
 
 Currently, TiDB Lightning supports:
 
-- The data source of the [Dumpling](/dumpling-overview.md), CSV or [Amazon Aurora Parquet](/migrate-aurora-to-tidb.md) exported formats.
+- The data source of the [Dumpling](/dumpling-overview.md), CSV or standard parquet file such as [Amazon Aurora Parquet](/migrate-from-aurora-using-lightning.md) exported formats.
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-overview.md:18]:
```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).
```

File: tidb-lightning/tidb-lightning-overview.md [hunk 2]
```diff
@@ -55,3 +55,5 @@ TiDB Lightning also supports using TiDB-backend for data import. In this mode, `
 - If you use TiDB Lighting together with TiDB:
 
     TiDB Lightning does not support importing `charset=GBK` tables to TiDB clusters earlier than v5.4.0.
+
+- When using parquet file to import, lightning currently only accepts the data types supported in Amazon Aurora Parquet.
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-overview.md:59]:
```suggestion
- For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.
```

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-overview.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Currently, TiDB Lightning can mainly be used in the following two scenarios:
 
 Currently, TiDB Lightning supports:
 
-- The data source of the [Dumpling](/dumpling-overview.md), CSV or [Amazon Aurora Parquet](/migrate-aurora-to-tidb.md) exported formats.
+- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-overview.md:18]:
```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-aurora-to-tidb.md).
```

**Review by sunzhaoyang — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**